src/main/java/org/isf/utils/log/LogUtil.java

```diff
@@ -43,8 +43,8 @@ public static String getLogFileAbsolutePath() {
 		Logger rootLogger = Logger.getRootLogger();
 		Appender appender = rootLogger.getAppender("RollingFile");
 
-		if (appender instanceof DailyRollingFileAppender) {
-			DailyRollingFileAppender rollingFileAppender = (DailyRollingFileAppender) appender;
+		if (appender instanceof RollingFileAppender) {
+			RollingFileAppender rollingFileAppender = (RollingFileAppender) appender;
```

Comment:
```suggestion
		if (appender instanceof RollingFileAppender rollingFileAppender) {	
```
Lines 46 and 47 can be replaced with this single line.   This is called a "Pattern variable" that was introduced in Java 16.